src/pgduckdb_node.cpp

```diff
@@ -66,8 +85,34 @@ ExecuteQuery(DuckdbScanState *state) {
 	auto &prepared = *state->prepared_statement;
 	auto &query_results = state->query_results;
 	auto &connection = state->duckdb_connection;
+	auto pg_params = state->params;
+	duckdb::vector<duckdb::Value> duckdb_params;
+	if (pg_params) {
+		for (int i = 0; i < pg_params->numParams; i++) {
+			ParamExternData *pg_param;
+			ParamExternData tmp_workspace;
```

Comment:
minor nits (constify upper bound, prefix increment and hoist loop variables):
```suggestion
		const auto n = pg_params->numParams;
		ParamExternData *pg_param = NULL;
		ParamExternData tmp_workspace;
		for (int i = 0; i < n; ++i) {
```